bitrise.yml

```diff
@@ -900,6 +908,14 @@ workflows:
         - connection: 'off'
         - itunescon_user: "$APPLE_ACCOUNT_ID"
         title: 'Nightly FirefoxBeta: Xcode Archive & Export for iOS'
+    - script@1:
+        title: Upload Nightly Symbols
+        inputs:
+        - content: |-
+            #!/usr/bin/env bash
+            set -x
+            ThirdParty/sentry-cli --auth-token "$SENTRY_AUTH_TOKEN" upload-dif \
+              --org mozilla --project firefox-ios "$BITRISE_DSYM_DIR_PATH"
```

Comment:
@yoanarios we should add this script to `SPM_Deploy_Beta_Only` as well

Reply:
https://github.com/mozilla-mobile/firefox-ios/pull/10105/files/b5373c272db9aab0bc9feec4f7d1617b7f754b07#diff-61d1ee831cb1df242161a7d23171914a1d5d628c7659b52db957d80a9394bd68R742

Reply:
done, thank you